src/releasers/python.ts

```diff
@@ -1,159 +0,0 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-import chalk = require('chalk');
-
-import {ReleasePR, ReleaseCandidate, PackageName} from '../release-pr';
-import {Update} from '../updaters/update';
-
-// Generic
-import {Changelog} from '../updaters/changelog';
-// Python specific.
-import {SetupPy} from '../updaters/python/setup-py';
-import {SetupCfg} from '../updaters/python/setup-cfg';
-import {PythonFileWithVersion} from '../updaters/python/python-file-with-version';
-import {ReleasePRConstructorOptions} from '..';
-import {
-  parsePyProject,
-  PyProject,
-  PyProjectToml,
-} from '../updaters/python/pyproject-toml';
-import {GitHubFileContents} from '../github';
-import {logger} from '../util/logger';
-
-const CHANGELOG_SECTIONS = [
```

Comment:
I'm curious where the default `CHANGELOG_SECTIONS` are defined now for Python? I used to refer to this page often to know which conventional commits refer to which sections.

Reply:
I also was curious about that, I found out that the default sections are defined in another package: https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-conventionalcommits/writer-opts.js#L179-L192